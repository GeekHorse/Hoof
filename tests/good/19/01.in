new right one done
new down two pause foo bar literal done cancel resume three done
new down four literal pause literal resume five done
most up
value
down
value
down
value
save
