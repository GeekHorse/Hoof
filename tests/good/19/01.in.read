new 
ok 
new 
paused 
resumed 
ok 
new 
ok 
ok 
ok   one 
ok 
ok   two three 
ok 
ok   four pause resume five 
ok 
