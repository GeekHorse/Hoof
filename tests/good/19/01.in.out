new right one done
new down two three done
new down four literal pause resume five done
